chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -243,6 +241,7 @@ impl PeerManagerActor {
                         async move {
                             loop {
                                 interval.tick(&clock).await;
+                                state.tier1_request_full_sync();
```

Comment:
was it a bug that it was missing from here before?

Reply:
No, it wasn't. Before this PR this was triggered elsewhere. Also we had a test which failed specifically because I removed triggering it elsewhere ;).